config/nav.yml

```
@@ -28,6 +28,7 @@ nav:
     - Installing:
       - About installing Knative: install/README.md
       - Install Knative using quickstart: install/quickstart-install.md
+      - Installing the Knative CLI: client/install-kn.md
```

```suggestion
      - Install the Knative CLI: client/install-kn.md
```
I think we should change this to "Install ..." to match the other items in this section. It looks odd being the only one that begins with "Installing".

---

Also, all the install Knative topics (quickstart, YAML, and Operator) used to be grouped together. Now we have the CLI install between the quickstart and YAML install. Maybe consider changing the ordering?